doc/03-multi-player.md

```diff
@@ -0,0 +1,42 @@
+Multi Player Mode
+======================
+
+
+This guide will walk you through running all the labs for a team of people.
+
+## 1. Prerequisites
+
+
++ You should probably skim through [the architecture outline](./01-architecture.md) first.
++ You should be able to setup the labs in [single player mode](./02-single-player.md).
+
+## 2. Initialization
+
+When provisioning for an actual training, the following is needed (to invite users):
+
+```(shell)
+make apps invite_users=1 env_csv=[CSV File with ssh keys] labs_json=labs_all.json prefix=[PREFIX] inviter_password=[Your Vendor web password] invite_rbac_policy_id=[Check the RBAC token in chrome dev tools] inviter_email=[your Vendor web email]
+```
+
+A csv example:
+```csv
+Timestamp,Source Email,name,pub_key,password,email,slug
+5/13/2021 18:02:25,participant.one@somecompany.com,Participant One,ssh-rsa public key particpant.one@somecompany.com,password,participant.one+[PREFIX]@somecompany.com,participant
+```
```

Comment:
Might be worth pointing out which fields are read, and link to the struct they're read into?